package.json

```diff
@@ -30,21 +30,21 @@
     "accepts": "~1.2.7",
     "content-disposition": "0.5.0",
     "content-type": "~1.0.1",
-    "cookie": "0.1.2",
+    "cookie": "0.1.3",
     "cookie-signature": "1.0.6",
     "debug": "~2.2.0",
     "depd": "~1.0.1",
-    "escape-html": "1.0.1",
+    "escape-html": "1.0.2",
     "etag": "~1.6.0",
     "finalhandler": "0.3.6",
-    "fresh": "0.2.4",
+    "fresh": "0.3.0",
     "merge-descriptors": "1.0.0",
     "methods": "~1.1.1",
-    "on-finished": "~2.2.1",
+    "on-finished": "~2.3.0",
     "parseurl": "~1.3.0",
-    "path-to-regexp": "0.1.3",
+    "path-to-regexp": "0.1.5",
     "proxy-addr": "~1.0.8",
-    "qs": "2.4.2",
+    "qs": "3.1.0",
```

Comment:
Unfortinately we cannot upgrade `qs` past 2.4.2, as there is a very user-visible breaking change.